### DISCUSSION TITLE
Edit action in View Resouce Page

### DISCUSSION THREAD

**Discussion by leandrogehlen** [filamentphp/filament#11927] [General]
Is there some way to add edit action to View page?
Example:

![image](https://github.com/filamentphp/filament/assets/1750751/d4741dda-b32b-45c8-9029-ed06fe61b5f9)

**Comment by leandrocfe** [comment]
add in the ViewPage

```php
use Filament\Actions;
protected function getHeaderActions(): array
{
    return [
        Actions\EditAction::make()
    ];
}
```
